example/src/lib/relay_client_environment.ts

```diff
@@ -25,7 +25,7 @@ let clientEnv: Environment | undefined;
 export function getClientEnvironment() {
   if (typeof window === 'undefined') return null;
 
-  if (clientEnv == null) {
+  if (clientEnv !== null && clientEnv !== undefined) {
```

Comment:
I think this change breaks the app — `clientEnv` will no longer be initialized.

Reply:
Yup, you are right now I tried the same change on my app it's working.

The issue was when you use the `==` it gives a lint issue. And I change it to === and it stopped working so I think it will be a case of a lot of people that this issue will occur. That's all.

And I opened one more MR #24  can you please check it. Thanks